### PR TITLE
add lib for inspect command

### PR DESCRIPTION
**Pull request by zhouying7780**
@kasturinarra Could you please help review ? thanks.

### CODE REVIEW

**Comment by pruan-rht** [comment]
@zhouying7780 does this apply for OCP < 4.4 as well? If that's the case we need to add that to the 4.1 yaml as well.

File: lib/rules/cli/4.4.yaml [hunk 1]
```diff
@@ -1136,6 +1136,13 @@
     :sync_config: --sync-config=<value>
     :type: --type=<value>
     :whitelist: --whitelist=<value>
+:oadm_inspect:
+  :cmd: oc adm inspect
+  :options:
+    :all_namespaces: --all-namespaces=<value>
```
Comment (akostadinov) [inline lib/rules/cli/4.4.yaml]: is this not a global option already?

**Comment by akostadinov** [comment]
@pruan-rht is correct. WIll let this merge in the mean time though

/lgtm

File: lib/rules/cli/4.4.yaml (hunk 1 above)
Comment (pruan-rht) [inline lib/rules/cli/4.4.yaml]: @zhouying7780  `all_namespaces` is a already defined, so I think we can take it out.

Comment (pruan-rht) [reply lib/rules/cli/4.4.yaml]: @zhouying7780 please ignore comment...all_namespaces is apparently not defined as a global option...perhaps it should.

Comment (akostadinov) [reply lib/rules/cli/4.4.yaml]: That was my initial reaction. But `oc options` does not return it. So it should not be in globals.

**Comment by liangxia** [comment]
> does this apply for OCP < 4.4 as well? If that's the case we need to add that to the 4.1 yaml as well.

@pruan-rht  We have the command in 4.1.yaml, just some option names are not consistent in 4.1 and 4.4. Fixed the inconsistent in https://github.com/openshift/verification-tests/pull/881
```
$ grep -r 'adm_inspect' *
features/registry/registry.feature:    When I run the :oadm_inspect admin command with:
lib/rules/cli/4.1.yaml::oadm_inspect:
lib/rules/cli/4.4.yaml::oadm_inspect:
```

**Pull request merged**